front/src/applications/stdcm/utils/formatStdcmConfV2.ts

```diff
@@ -118,6 +123,22 @@ export const checkStdcmConf = (
       })
     );
   }
+  const startDatetime = new Date(`${startTime}`);
+  if (
+    searchDatetimeWindow &&
```

Comment:
Use a else if (), like this you know that startTime is not null
```suggestion
  } else if (searchDatetimeWindow) {
    const startDatetime = new Date(startTime);
  if (
```